pkg/controller/namespace/namespace_controller_test.go

```diff
@@ -91,60 +91,51 @@ cluster_id="42"
 
 archive=$(mktemp)
 
-if [[ "${FAILURE_POLICY}" == "fail" ]]; then
-	fail_code=1
-fi
-
-# Work around to use installer URL until we have the image.
-#if [[ "${INSTALLER_URL}" != "" ]]; then
-
 if [[ "${INSTALLER_URL}" != "" ]]; then
 	installer_url="${INSTALLER_URL}"
-fi
 
-curl_params=(
-	"--silent"
-	"--output" "${archive}"
-	"${installer_url}"
-)
-
-if [[ "${INSTALLER_URL}" == "" ]]; then
-	curl_params+=("--header" "Authorization: Api-Token ${paas_token}")
-fi
-
-if [[ "${skip_cert_checks}" == "true" ]]; then
-	curl_params+=("--insecure")
+	if [[ "${FAILURE_POLICY}" == "fail" ]]; then
+		fail_code=1
+	fi
```

Comment:
Please put this block outside the upper `if`. Even if it's unused now, the fail_code can also apply to other non-download operations.